src/apps/explorer/components/TokensTableWidget/index.tsx

```diff
@@ -156,3 +156,11 @@ export const TokensTableWidget: React.FC<Props> = () => {
     </TableWrapper>
   )
 }
+
+function undefinedToNull(n: number | undefined | null): number | null {
+  if (n === undefined) {
+    return null
+  }
+
+  return n
```

Comment:
What about:

```suggestion
  return n ?? null
```

Reply:
You could actually replace the `|| null` with `?? null` and not need this function.